### PR TITLE
Add HoloPlay to Media/Audio Streaming

### PR DESCRIPTION
**Pull request by stephane-r**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [ ] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.

### CODE REVIEW

File: software/holoplay.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: "HoloPlay"
+website_url: "https://app.holoplay.io"
+source_code_url: "https://github.com/stephane-r/holoplay-pwa"
+description: "Web app using Invidious API for listening Youtube audio source."
+licenses:
+  - MIT
+platforms:
+  - Docker
+  - React
```
Comment (KieranRobson) [inline software/holoplay.yml:9]:
```suggestion
platforms:
  - Nodejs
  - Docker
```

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Quick change to be made.

**Review by kokomo123 — APPROVED** [review]
Since the proper adjustments have been made, I believe this can be approved fully.  It's mature, being actively committed to, etc. Thanks for your contribution

**Comment by nodiscc** [comment]
>  Any software project you are adding was first released more than 4 months ago.

As far as I can tell the first release was created on July 13, so I don't think this currently qualifies.

**Comment by kokomo123** [comment]
I thought it was time of first commit, not first release. Oops.

**Comment by KieranRobson** [comment]
> I thought it was time of first commit, not first release. Oops.

I use first commit if software does not have a release, I seemed to have also missed this.

**Pull request closed without merging**